File: Zig/square.zig
const std = @import("std");
const print = std.debug.print;


pub fn main() !void {
    const val = 10;
    const result = sqr(val);
    print("Result: {}\n", .{result});
}

fn sqr(val: i32) i32 {
    return val * val;
}

File: C/Practice/square.c
#include <stdio.h>

int sqr(int val);

int main()
{
    int val = 10;
    int result = sqr(val);

    printf("Result: %d\n", result);

    return 0;
}

// Function definition
int sqr(int val)
{
    return val * val;
}
